### ISSUE TITLE
why is oval:org.mitre.oval:def:12063 (OpenOffice < 3.3) not included in W8/W10 Definitions?

### ISSUE THREAD

**Issue by Memnarch** [CISecurity/OVALRepo#1601]
I just wondered, why the Definitions is only available up to W7 definitions? I'm aware that this is q uite old OO Version, but what blocks OO (older than 3.3) from existing on W8/W10 machines?

Is it more advisable to run Definitionfiles tailored/filtered for a specify OS (W7/W8/W10) or should one simply use the Family Definitionfile for his platform? (in my case Windows)

**Comment by DavidRies** [comment]
Hi @Memnarch, 

I don't know anything whether OpenOffice can run on W8, etc. or whether this definition would be accurate on those platforms.

However, the content author put the specific OS versions that this definition was written to apply to in the definition/metadata. So, I would assume this definition was not tested on W8/10/etc.. It's possible that the definition logic would need to be altered to be accurate on W8/10/etc. and/or the vulnerability may be inapplicable to those platforms.

Generally speaking I would recommend running definitions that are:
- written for the specific OS you are assessing
- OR, applicable to the family of the OS you are running and not OS-specific (no platforms defined)

Does that make sense?

-David

**Comment by Memnarch** [comment]
Makes sense. Just one more question: Those without Platform, are they included in all Definitions?

**Comment by DavidRies** [comment]
I am not sure which "all definitions" package you mean...?

**Comment by Memnarch** [comment]
I mean, when a definition has no Platform, is it included in all platform specific definitionfiles, like the W7/W8/W10 Definitions? SO when i download the W10-Definitionfile, does it include not just Definitions with W10 as Platform but all other definitions without any platform, too?

**Comment by DavidRies** [comment]
That's a good question. 

I believe CIS-CAT uses the [scripts/build_oval_definitions_file.py](https://github.com/CISecurity/OVALRepo/blob/master/scripts/) to create the packages. I wrote that script, :), but I didn't remember so I just checked and it looks like it just includes definitions based on the provided filters. So, if you build a package for a family/platform it will include only definitions that explicitly reference them. 

However, I may just be overthinking this. I'm not sure it's necessary to include definitions that don't reference a platform. I spot checked some MS Office CVEs and they all seemed to include platforms [like this one](https://github.com/CISecurity/OVALRepo/blob/9d5b9104e0c879f97f5a63382b4402a9b743e29b/repository/definitions/vulnerability/oval_org.cisecurity_def_3890.xml).

Since we rely on the platform tag for packaging, this should probably be a repository best practice.
